### ISSUE TITLE
Admin panel crashes when signing in

### ISSUE THREAD

**Issue by kuyugama** [vsdudakov/fastadmin#28]
I've already opened the same issue, but the "correction" doesn't work

The account ID that is passed to token is always a string

My suggestion is to change the code https://github.com/vsdudakov/fastadmin/blob/f618ac98f8f50f5cddf835139ca626fe6c2e7/fastadmin/api/service.py#L121:
`"user_id": str(user_id),`
To:
`"user_id": user_id,`

And add this code before return:
```
if isinstance(user_id, UUID) :
    user_id = str(user_id)
```

**Comment by vsdudakov** [comment]
`  File "C:\Users\Владислав\PycharmProjects\mitsu_api\virtualenv\Lib\site-packages\fastadmin\api\frameworks\fastapi\api.py", line 329, in configuration
    return await api_service.get_configuration(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Владислав\PycharmProjects\mitsu_api\virtualenv\Lib\site-packages\fastadmin\api\service.py", line 414, in get_configuration
    current_user_id = await get_user_id_from_session_id(session_id)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Владислав\PycharmProjects\mitsu_api\virtualenv\Lib\site-packages\fastadmin\api\service.py", line 62, in get_user_id_from_session_id
    if not user_id or not await admin_model.get_obj(user_id):
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Владислав\PycharmProjects\mitsu_api\virtualenv\Lib\site-packages\fastadmin\models\base.py", line 415, in get_obj
    obj = await self.orm_get_obj(id)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Владислав\PycharmProjects\mitsu_api\virtualenv\Lib\site-packages\fastadmin\models\orms\sqlalchemy.py", line 314, in orm_get_obj
    return await session.get(self.model_cls, id)`

**Comment by vsdudakov** [comment]
Investigation:
 - we get configuration after sign in to render UI
 - we get current_user_id from session (jwt token which we store in http only cookies)
 current_user_id = await get_user_id_from_session_id(session_id) (fastadmin\api\service.py, line 447)

In get_user_id_from_session_id:
```
    user_id = convert_id(user_id)
    if not user_id or not await admin_model.get_obj(user_id):
        return None
```

After a fix in 0.1.37 we have convert_id and get_obj should receive int | UUID not string

get_obj -> orm_get_obj (SQL Alchemy orm)

```
    async def orm_get_obj(self, id: UUID | int) -> Any | None:
        sessionmaker = self.get_sessionmaker()
        async with sessionmaker() as session:
            return await session.get(self.model_cls, id)
```

**Comment by vsdudakov** [comment]
I cannot reproduce it on my side. I will try cover more this functionality by tests and hopefully I will find the issue

**Comment by vsdudakov** [comment]
My suggestion is to change the code https://github.com/vsdudakov/fastadmin/blob/f618ac98f8f50f5cddf835139ca626fe6c2e7/fastadmin/api/service.py#L121:
"user_id": str(user_id),
To:
"user_id": user_id,

And add this code before return:

if isinstance(user_id, UUID) :
    user_id = str(user_id)
    
    
I will try the solution. Thank you! But I think we will have an issue to encode UUID values into jwt

**Comment by vsdudakov** [comment]
https://github.com/vsdudakov/fastadmin/compare/0.1.37...0.1.38
@kuyugama  Provided your solution in 0.1.38. Hope it will help.

**Comment by kuyugama** [comment]
> Investigation:
> 
> * we get configuration after sign in to render UI
> * we get current_user_id from session (jwt token which we store in http only cookies)
>   current_user_id = await get_user_id_from_session_id(session_id) (fastadmin\api\service.py, line 447)
> 
> In get_user_id_from_session_id:
> 
> ```
>     user_id = convert_id(user_id)
>     if not user_id or not await admin_model.get_obj(user_id):
>         return None
> ```
> 
> After a fix in 0.1.37 we have convert_id and get_obj should receive int | UUID not string
> 
> get_obj -> orm_get_obj (SQL Alchemy orm)
> 
> ```
>     async def orm_get_obj(self, id: UUID | int) -> Any | None:
>         sessionmaker = self.get_sessionmaker()
>         async with sessionmaker() as session:
>             return await session.get(self.model_cls, id)
> ```

Converting id to a number after it has been converted to a string is a bad decision. Because you can store id already in the desired data type

**Comment by vsdudakov** [comment]
Agreed but it doesn't work for UUID , could you please check 0.1.38?

**Comment by vsdudakov** [comment]
In case it wrks I will clean up code  and remove convert_id keep it only for UUID may be
